pkg/genericapiserver/genericapiserver.go

```diff
@@ -269,6 +269,10 @@ type GenericAPIServer struct {
 	enableOpenAPISupport   bool
 	openAPIInfo            spec.Info
 	openAPIDefaultResponse spec.Response
+
+	// PostStartHooks is called in a go func after the server has started listening.
```

Comment:
> "in a go func"?

Yeah, I'll have to try to clarify.  There's no ordering guarantee between anything in the array.  It just loops, logically:

```go
for _, hook := range hooks{
    go hook()
}
```